kong/plugins/prometheus/exporter.lua

```diff
@@ -31,6 +31,65 @@ package.loaded['prometheus_resty_counter'] = require("resty.counter")
 local kong_subsystem = ngx.config.subsystem
 local http_subsystem = kong_subsystem == "http"
 
+
+-- should we introduce a way to know if a plugin is configured or not?
+local instance_update_register, has_instance do
+  local plugin_name = "prometheus"
+  local instance_count = 0
+
+
+  local function initialize_instance_counters()
+    instance_count = 0
+    for plugin, err in kong.db.plugins:each() do
+      if err then
+        kong.log.crit("could not obtain list of plugins: ", err)
+        return nil, err
+      end
+
+      if plugin.name == plugin_name then
```

Comment:
You need to consider whether `enabled = true` as well.

Reply:
done.